<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ftasettingdialog</class>
 <widget class="QDialog" name="ftasettingdialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FTA Setting</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>441</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Time to Arrival</string>
   </property>
   <widget class="QRadioButton" name="fromshoworbit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>391</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Get Elapsed Time of Show Orbit window</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="directinput">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>391</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Specify Time to Arrival</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="timetoarrival">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>60</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Console</family>
     </font>
    </property>
    <property name="text">
     <string>100.00000</string>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>60</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Time to Arrival (days)</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="cancel_button">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>440</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="ok_button">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>440</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>441</width>
     <height>291</height>
    </rect>
   </property>
   <property name="title">
    <string>Precise Targeting</string>
   </property>
   <widget class="QRadioButton" name="selTargetcenter">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>391</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Center of Target</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="selBplanecoord">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>391</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>B-plane Coordinates</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="selOLcoord">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>391</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Orbit Local Coordinates</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="Bplanecoords">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>261</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>offset distance (km)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Brangeedit">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>10</y>
       <width>91</width>
       <height>19</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Console</family>
      </font>
     </property>
     <property name="text">
      <string>0.0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>beta (deg)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="betaedit">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>91</width>
       <height>19</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Console</family>
      </font>
     </property>
     <property name="toolTip">
      <string>Angle beta on B-plane</string>
     </property>
     <property name="text">
      <string>0.0</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="OLcoords">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>190</y>
      <width>261</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>offset distance (km)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>phi (deg)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>elv (deg)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="rangeedit">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>10</y>
       <width>91</width>
       <height>19</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Console</family>
      </font>
     </property>
     <property name="text">
      <string>0.0</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="phiedit">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>91</width>
       <height>19</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Console</family>
      </font>
     </property>
     <property name="toolTip">
      <string>Angle phi on Orbit Local coordinate</string>
     </property>
     <property name="text">
      <string>0.0</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="elvedit">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>50</y>
       <width>91</width>
       <height>19</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Console</family>
      </font>
     </property>
     <property name="toolTip">
      <string>Angle elv on Orbit Local coordinate</string>
     </property>
     <property name="text">
      <string>0.0</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
